dependency_manager/src/edm_tool/edm.py

```diff
@@ -1490,7 +1510,14 @@ def main_handler(args):
     workspace = EDM.parse_workspace_directory(workspace_dir)
     checkout = EDM.checkout_local_dependencies(workspace, args.workspace, dependencies)
 
-    # modify dependencies from environment variable
+    # modify dependencies from environment variables
```

Comment:
I missunderstood the comment when first reading, maybe:

```suggestion
    # Apply modifications from environment variables to the dependencies
```